app/models/katello/concerns/pulp_database_unit.rb

```diff
@@ -69,6 +69,7 @@ def import_all(uuids = nil, options = {})
 
       def import_for_repository(repository, force = false)
         ids = content_unit_class.ids_for_repository(repository.pulp_id)
+        ids -= repository.rpms.map(&:uuid) if self == Katello::Rpm
```

Comment:
Errata and package groups can be updated, so we can't completely skip it for them.  Here we are assuming that rpms can never be updated (and currently that is the behavior in pulp).  We could probably do this for puppet modules, but that generally doesn't have nearly as big of a performance penalty.